container/container.go

```diff
@@ -88,7 +87,7 @@ type Container struct {
 	MountPoints            map[string]*volume.MountPoint
 	HostConfig             *containertypes.HostConfig `json:"-"` // do not serialize the host config in the json, otherwise we'll make the container unportable
 	ExecCommands           *exec.Store                `json:"-"`
-	DependencyStore        agentexec.DependencyGetter `json:"-"`
+	DependencyStore        interface{}                `json:"-"`
```

Comment:
Will it still be allocated? I think these stuff are also defined in the protobuff. at containerd/containerd/api/grpc/types/api.pb.go